Sources/Brave/Frontend/Browser/Playlist/Managers & Cache/PlaylistCacheLoader.swift

```diff
@@ -36,7 +36,7 @@ class LivePlaylistWebLoader: UIView, PlaylistWebLoader {
     }, type: .private
   ).then {
     $0.createWebview()
-    $0.setScript(script: .playlistMediaSource, enabled: Preferences.Playlist.webMediaSourceCompatibility.value)
```

Comment:
Should we remove the preference entirely ?

Reply:
No. We should keep it just in case. Users can select it from settings which can allow them to downgrade quality if they wish, and use a different API. We can remove it in the future if it becomes useless.